Scripts/CMakeLists.txt

```diff
@@ -6,60 +6,65 @@ source_group("Python" FILES ${GameScripts})
 source_group("SDL" FILES ${GameSDL})
 source_group("Ages" FILES ${GameData})
 
-add_custom_target(Scripts ALL
-                  COMMENT "Copying engine scripts..."
-
-                  # Copy core engine files
-                  COMMAND ${CMAKE_COMMAND} -E copy_directory "${CMAKE_CURRENT_SOURCE_DIR}/dat" "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/dat"
-                  COMMAND ${CMAKE_COMMAND} -E copy_directory "${CMAKE_CURRENT_SOURCE_DIR}/Python" "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/python"
-                  COMMAND ${CMAKE_COMMAND} -E copy_directory "${CMAKE_CURRENT_SOURCE_DIR}/SDL" "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/SDL"
-
+add_custom_target(Scripts
                   SOURCES ${GameScripts} ${GameSDL} ${GameData}
 )
```

Comment:
Is this target actually still needed?

Reply:
No, but I think @Deledrius liked being able to see the game scripts in VS.

Reply:
Ah, right, I hadn't considered the IDE integration...

Reply:
If you feel it's extraneous, I won't argue the point.  The gamescripts straddle the line between source and content in annoying ways.  I just felt that if we have text source in the tree, and we're doing things with them, it would be nice to have them be accessible and organized in the editor, if nothing else but for transparency.